### DISCUSSION TITLE
Native Azure Portal feature

### DISCUSSION THREAD

**Discussion by Karl-WE** [Azure/ipam#293] [Ideas]
As this solution is under very active development and many requests, I would appreciate a native feature into the Portal.

What are reasons, except community contributions, that it is a standalone optional feature? 

The idea could be to have it natively in the portal and retain the community contributions by linking to this repo via the usual get started tab.

**Comment by DCMattyG** [comment]
Whilst I completely agree with you @Karl-WE, I think it's important to remember that this is an open-source project and we don't work directly for the Azure product groups whom would be responsible for bringing a solution like as a 1st party solution in Azure.

That said we have indeed been actively engaged with the Azure Networking team to bring a 1st party IPAM solution into Azure Virtual Network Manager. This solution is currently in Private Preview and was announced during the most recent Ignite.

We're always happy to hear feedback such as this, but there are other forums for formally requesting Azure features/changes and that generally start here: https://feedback.azure.com/

Hope that help provide a bit of insight for you!

**Reply by Karl-WE** [reply]
Yes that helped a lot. Thanks @DCMattyG.
